package.json

```diff
@@ -29,6 +30,10 @@
       {
         "command": "extension.runSelectedAsQuery",
         "title": "BigQuery: Run selected text as Query"
+      },
+      {
+        "command": "extension.dryRun",
+        "title": "BigQuery: Dry run Query"
```

Comment:
"Run as Dry Run"

Reply:
Done (I played with the wording here a bit - nothing seems great!)